src/plus/integrations/authentication/integrationAuthentication.ts

```diff
@@ -16,7 +16,7 @@ import {
 } from '../providers/models';
 import type { ProviderAuthenticationSession } from './models';
 
-interface StoredSession {
+export interface StoredSession {
```

Comment:
We no longer need this to be exported since we removed its usage in `gitlab.ts`

Reply:
fixed